sites/footers/reports/a11y-1.json

```diff
@@ -0,0 +1,905 @@
+      "earl:test": "WCAG20:three-flashes-or-below-threshold",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G19",
+        "WCAG21Tech:general/G176",
+        "WCAG21Tech:general/G15"
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 2.4.1 Bypass Blocks (Level A)",
+      "earl:test": "WCAG20:bypass-blocks",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G1",
+        "WCAG21Tech:general/G123",
+        "WCAG21Tech:general/G124"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 2.4.2 Page Titled (Level A)",
+      "earl:test": "WCAG20:pause-stop-hide",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": ["WCAG21Tech:general/G88", "WCAG21Tech:failures/F25"],
+      "earl:mode": "earl:semiAuto",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 2.4.3 Focus Order (Level A)",
+      "earl:test": "WCAG20:focus-order",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G59",
+        "WCAG21Tech:failures/F44",
+        "WCAG21Tech:failures/F85"
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 2.4.4 Link Purpose (In Context) (Level A)",
+      "earl:test": "WCAG20:link-purpose-in-context",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G91",
+        "WCAG21Tech:general/G189",
+        "WCAG21Tech:general/G53",
+        "WCAG21Tech:failures/F63",
+        "WCAG21Tech:failures/F89"
+      ],
+      "earl:mode": "earl:semiAuto",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 2.4.5 Multiple Ways (Level AA)",
+      "earl:test": "WCAG20:multiple-ways",
+      "outcome": "Inapplicable",
+      "earl:outcome": "earl:inapplicable",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G125",
+        "WCAG21Tech:general/G64",
+        "WCAG21Tech:general/G63",
+        "WCAG21Tech:general/G161",
+        "WCAG21Tech:general/G126",
+        "WCAG21Tech:general/G185"
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 2.4.6 Headings and Labels (Level AA)",
+      "earl:test": "WCAG20:headings-and-labels",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": ["WCAG21Tech:general/G130", "WCAG21Tech:general/G131"],
+      "earl:mode": "earl:semiAuto",
```

Comment:
Just for my knowledge, what tool have you used for this SC evaluation?

Note for myself, we will need to adjust our JSON-LD reporting for the property "assertedBy" to allow the addition of a reference to the tools being used.

Reply:
The AT tool do have its own result item identified that was asserted by that tool and it is also defined in the assessment assessor via the property `earl:assertedBy`.